src/lib/commands/auth.ts

```diff
@@ -1,11 +1,13 @@
 import { commands, env, ExtensionContext, Uri } from 'vscode';
 import { COMMANDS } from '../constants';
 
-export const registerCommands = (context: ExtensionContext) => {
+export const registerCommands = (
+  context: ExtensionContext,
+  rootPath: string
+) => {
   context.subscriptions.push(
     commands.registerCommand(COMMANDS.AUTHORIZE_ACCOUNT, async () => {
-      const authUrl =
-        'https://app.hubspot.com/l/personal-access-key/auth/vscode';
+      const authUrl = `https://app.hubspot.com/l/personal-access-key/auth/vscode${rootPath}`;
```

Comment:
I can't fully test the flow on Windows but since `rootPath` uses `fsPath`, this path contains the windows style path which looks like `.../auth/vscodec:\folder` which causes a 404

Reply:
Switched to using `path` instead of `fsPath` and also added a fix for the value of `rootPath` that is returned from personal-access-key-ui, which is something like `/C:/Folder`. The fix slices off the leading `/`.

Let me know if you see any issues with this.